tests/insert_reference/insert_reference.expected

```diff
@@ -1,2 +1,2 @@
 Add reference(s):
-* target.html, l3 --> source.ml, l3, 1 line high
+* target.html, l10 --> source.ml, l3, 1 line high
```

Comment:
I would write `L10` instead of `l10`.

This said, I would prefer if we used a standard format like file URI for instance

https://tools.ietf.org/html/draft-ietf-appsawg-file-scheme-03

or GCC error position format.

Reply:
The problem is that it isn't an error.... It's just a print to show where references are inserted... So do I need to use this error format ?

Reply:
I use GCC format.